content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,123 @@
+# data status
+
+Show changes in data tracked by DVC between the last Git commit, the metafiles
+and the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc data status [-h] [-q | -v]
+                       [--granular] [--unchanged]
+                       [--untracked-files [{no,all}]]
+                       [--json]
+```
+
+## Description
+
+Shows changes that have differences between the last Git commit (`HEAD`) and
```

Comment:
This is the same as `Show changes in data tracked by DVC between the last Git commit, the metafiles
and the workspace.` ... It doesn't give any new information to my mind

Reply:
I mean the whole paragraph. No need to artificially expand on the same stuff. Or put some meaningful intro that would be useful. E.g. we can describe high level one more time why Git status is not enough, etc ... (where does the need for this comes from at all)